src/common.ts

```diff
@@ -19,3 +21,84 @@ export const getFunds = async (address: string, url?: string) => {
     );
   }
 };
+
+
+/**
+ * Sends funds to the specified address
+ * @param hre Hardhat Runtime Environment
+ * @param toAddress Address to send funds to
+ * @param amount Amount to send in ETH (default: 10)
+ * @returns Transaction receipt or null if failed
+ */
+export async function sendFunds(
```

Comment:
If we're going to export this function as a utility, we need to give it a more descriptive name like "localcofheFundAccount"